Sources/PkgConfig/PkgConfig.swift

```diff
@@ -150,11 +149,11 @@ struct PkgConfigParser {
     }
     
     private mutating func parseKeyValue(line: String) throws {
-        if line.hasPrefix("Requires: ") {
```

Comment:
I think this would be more simply written as:
```swift
precondition(line.characters.contains(":"))
let (key, value) = line.split(around: ":")
```
and then a switch on key.